From 5342675904c43e713e76b85c59c0e6caf99befa9 Mon Sep 17 00:00:00 2001
From: Dimitrij <Dima-73@inbox.lv>
Date: Sun, 27 Nov 2022 16:43:38 +0200
Subject: [PATCH] [EventView] add stream service in add/remove timer (#3569)

---
 lib/python/Screens/EventView.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/python/Screens/EventView.py b/lib/python/Screens/EventView.py
index c80e70f875f..381818800bc 100644
--- a/lib/python/Screens/EventView.py
+++ b/lib/python/Screens/EventView.py
@@ -28,7 +28,7 @@ def __init__(self, event, Ref, callback=None, similarEPGCB=None, parent=None):
 		self.similarEPGCB = similarEPGCB
 		self.cbFunc = callback
 		self.currentService = Ref
-		self.isRecording = (not Ref.ref.flags & eServiceReference.isGroup) and Ref.ref.getPath()
+		self.isRecording = (not Ref.ref.flags & eServiceReference.isGroup and Ref.ref.getPath()) and "%3a//" not in Ref.ref.toString()
 		self.event = event
 		self["Service"] = ServiceEvent()
 		self["Event"] = Event()
